module/docs/entra-powershell-beta/Microsoft.Graph.Entra.Beta/Get-EntraBetaGroupAppRoleAssignment.md

```diff
@@ -31,17 +31,20 @@ Get-EntraBetaGroupAppRoleAssignment
 ```
 
 ## Description
-The Get-EntraBetaGroupAppRoleAssignment cmdlet gets a group application role assignment in Microsoft Entra ID.
+
+The `Get-EntraBetaGroupAppRoleAssignment` cmdlet gets a group application role assignment in Microsoft Entra ID. Specify the `ObjectId` parameter to get a group application role assignment.
 
 ## Examples
 
 ### Example 1: Retrieve application role assignments of a group
+
 ```powershell
+Connect-Entra -Scopes 'Directory.Read.All'
 $GroupId = (Get-EntraBetaGroup -Top 1).ObjectId
 Get-EntraBetaGroupAppRoleAssignment -ObjectId $GroupId
 ```
 
-```output
+```Output
 ObjectId                                    ResourceDisplayName                 PrincipalDisplayName
 --------                                    -------------------                 --------------------
 MSVrBV4APk--eAGnHqMKBLflsQG3rU1EmDFKvgra41I Microsoft Device Management Checkin Ask HR
```

Comment:
Kindly remove this part:

The first command gets the object ID of a group by using the [Get-EntraBetaGroup](./Get-EntraBetaGroup.md) cmdlet.
The command stores the ID in the $GroupId variable.

Reply:
Changes done